internal/repos/importer.go

```diff
@@ -204,3 +215,54 @@ func (i *importer) handleDeletedRepos(ctx context.Context, tx pgx.Tx, repos []*g
 
 	return err
 }
+
+func (i *importer) handleDefaultSyncs(ctx context.Context, tx pgx.Tx, defaultSyncs []string, impID uuid.UUID) error {
+	var err error
+	var repoIDs []uuid.UUID
+
+	i.logger.Info().Msgf("starting to insert default syncs for import %s", impID)
+
+	if repoIDs, err = i.db.WithTx(tx).GetAllReposId(ctx, impID); err != nil {
+		i.logger.Err(err).Msgf("failed to retrieve repoids for import %s", impID)
+		return err
+	}
+
+	for _, repoID := range repoIDs {
+		if err = i.insertSyncInQueue(ctx, tx, defaultSyncs, repoID); err != nil {
+			i.logger.Err(err)
+			return err
+		}
+
+	}
+
+	i.logger.Info().Msgf("inserting of default syncs completed for import %s", impID)
+
+	return err
+}
+
+func (i *importer) insertSyncInQueue(ctx context.Context, tx pgx.Tx, defaultSyncs []string, repoID uuid.UUID) error {
+	var err error
+	var syncExistance bool
+	//testing methods that will be removed after draft
+	defaultSyncs = append(defaultSyncs, "GIT_COMMITS")
+	defaultSyncs = append(defaultSyncs, "GIT_COMMIT_STATS")
```

Comment:
@patrickdevivo  @amenowanna   guys please this line will be the line test  locally  the default syncs we want , will be removed after draft phase  along with the addition of the if statement for these logic to kick off